benchmarks/operator_benchmark/pt/fill_test.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import operator_benchmark as op_bench
+import torch
+
+"""Microbenchmark for Fill_ operator."""
+
+
+fill_short_configs = op_bench.cross_product_configs(
+    N=[10, 1000],
+    device=['cuda', 'cpu'],
+    dtype=[torch.int8, torch.uint8, torch.int16, torch.int32, 
+           torch.int64, torch.half, torch.float, torch.double],
```

Comment:
Can we also add bool here? Sorry I didn't add it in my original version.

Reply:
Added.

Reply:
This should be `torch.bool` not `torch.ByteTensor`...

Reply:
Changed in #22913 

On a different note, were you able to run the benchmark on your machine?

Reply:
Yes, I was able to run the benchmark (only on Python 2). But I found the instructions a bit incomplete (as in `benchmarks/operator_benchmark/README.md`:

- `python -m pt.xxx` should be run in `benchmarks/operator_benchmark/`, otherwise `pt` would not be found.
- `protobuf` should be installed (`pip install protobuf`).
- Python 2 must be used.

Reply:
Thanks for the feedback. I will modify the README accordingly so others won't encounter the same issue.